src/components/pages/Categories/PostList.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="category">
+    <div class="category_post">
+      <app-category-post
+        :access="access"
+        :error-message="errorMessage"
+      />
+    </div>
```

Comment:
[imo]

カスタムタグに関しても class の付与ができます。
不要なタグが削除できるので以下のように修正してみてください。

```suggestion
    <app-category-post
      :access="access"
      :error-message="errorMessage"
      class="category_post"
    />
```

Reply:
こちら修正しました。